README.md

```diff
@@ -1,11 +1,11 @@
 # Lens Desktop Core ("OpenLens")
 
 [![Build Status](https://github.com/lensapp/lens/actions/workflows/test.yml/badge.svg)](https://github.com/lensapp/lens/actions/workflows/test.yml)
-[![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-blue.svg?logo=slack&longCache=true&style=flat)](https://k8slens.dev/slack.html)
+[![Explore our Forums](https://upload.wikimedia.org/wikipedia/commons/1/17/Discourse_icon.svg)](https://forums.k8slens.dev)
```

Comment:
Is there something fishy here? The readme at https://github.com/lensapp/lens/tree/replace-slack-links is bizarre:
![Screen Shot 2023-02-24 at 4 10 01 PM](https://user-images.githubusercontent.com/40840436/221294224-222efac3-422a-4fd4-bfb4-a14323546cfe.png)